docs/API/InstrumentFormat.md

```diff
@@ -18,6 +23,37 @@ this standard without breaking backwards compatibility. Instruments MAY implemen
 the instrument format described below and the accompagnying rules format in the same
 JSON object, or they MAY be used independently.
 
+Language specifications can be added for fields that display text (LongName, Description, Values, Labels) in the following format:
```

Comment:
I don't think it's a good idea to have fields that can be "either format A or B". It should simply be standardized in 1 way that supports languages (you can always only implement a single language if you're not multilingual.)

Reply:
I'm not sure what you mean entirely but basically if you only want to implement 1 language without a language code thats completely possible (I do need to make a slight change to a single script but that was for a CAP requirement).

Reply:
I mean you shouldn't say "You can either use text *or* this format for fields x, y, and z", you should just say "This is the format for fields x, y, and z" for those fields and not try and support 2 different ways of doing the same thing..

(If that's not what you meant, then the phrasing of the paragraph is confusing and probably needs to be cleared up.)